CONTRIBUTING.adoc

```diff
@@ -0,0 +1,90 @@
+
+* The workshop is released under the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/blob/master/LICENSE[Apache 2.0 License]. Any code you submit will be released under this license. If you are contributing a large/substantial enhancement, you may be asked to sign a link:https://github.com/aws/aws-cla[Contributor License Agreement (CLA)].
+
+* For anything but very small or quick changes, you should always start by checking the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page to see if the work is already being done by another person.
+
+[NOTE]
+If you're working on a bug fix, check to see if the bug has already been reported. If it has but no one is assigned to it, ask one of the maintainers to assign it to you before beginning work.  If you're confident the bug hasn't been reported yet, create a new link:#bug-reports[Bug Report] then ask to be assigned to it.
+
+[NOTE]
+If you are thinking about adding entirely new functionality, open an link:#enhancements[Enhancements] or link:https://gitter.im/aws-samples/aws-workshop-for-kubernetes[ping] the maintainers to ask for feedback first before beginning work; again this is to make sure that no one else is already working on it, and also that it makes sense to be included in the workshop.
+
+=== Pull Request Readiness
+Before submitting your pull request, refer to the pull request readiness checklist below:
+
+* [ ] Includes tests to exercise the new behavior
+* [ ] Code is documented, especially public and user-facing constructs
+* [ ] Git commit message is detailed and includes context behind the change
+* [ ] If the change is related to an existing Bug Report or Enhancement, the issue number is referenced
+
+[NOTE]
+Some changes have additional requirements. Refer to the section below to see if your change will require additional work to be accepted.
+
+=== Getting Your Pull Request Merged
+All Pull Requests must be reviewed and approved by at least two other contributors or one maintainer before it can be merged in. Additionally, maintainers will strive to not merge their own pull requests unless 72 hours has passed, though extenuating circumstances may apply. The members only have limited bandwidth to review Pull Requests so it's not unusual for a Pull Request to go unreviewed for a few days, especially if it's a large or complex one. If, after a week, your Pull Request has not had any engagement from the workshop team, feel free to link:https://gitter.im/aws-samples/aws-workshop-for-kubernetes[ping] a member to ask for a review.
+
+If your branch has more than one commit when it's approved, you may also be asked to link:https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History[squash] them into a single commit before it is merged in.
+
+== Getting in Contact
+* Come chat with us on link:https://gitter.im/aws-samples/aws-workshop-for-kubernetes[Gitter]!
```

Comment:
Lets use the aws-developers slack channel instead.

Reply:
I think it's important to use both.  Gitter is widely accepted by the GitHub community and is a _very_ simple way of beginning a chat (whereas the Slack channels do require some registration and validation).

I did add the mention of the Slack channels.